#173 add support for a separate final image in containerize-dependencies (#174)
TechnologyBrewery/habushu: README.md

The plugin will automatically inject logic for building and using this virtual environment into a pre-existing Dockerfile
by default. The `dockerfile` configuration must be set to the target Dockerfile. To disable the Dockerfile update
altogether, set the `updateDockerfile` configuration to `false`. Because the final image must match the image that builds

> the virtual environment, Habushu defaults to using `python:3.11` to build the virtual environment and  `python:3.11-slim`

Reviewer: `Because the final image must match the image that builds the virtual environment`. This is no longer the case right?

Author: Reworded this section more significantly to put the problem of compatible venv up front.  Made this part more clear as a result.

as the final image that packages/runs the virtual environment. This can be customized with the `dockerBuilderBase`,

> `dockerFinalBase`, and `dockerUser` configurations, but care must be taken to ensure the builder image platform is
> compatible with the final image platform so that the virtual environment is compatible.

Reviewer 2: A: doesn't have to be exactly this, but the current working just felt like it was using the word compatible too much
```suggestion
`dockerFinalBase`, and `dockerUser` configurations, but care must be taken to ensure the virtual environment is compatible with both the builder image and final image platforms.
```

Author: Ah I didn't even notice I used the word twice. I'll clean it up.
